xwiki-platform-core/xwiki-platform-annotation/xwiki-platform-annotation-test/xwiki-platform-annotation-test-docker/src/test/it/org/xwiki/annotation/test/ui/AnnotationsIT.java

```diff
@@ -168,4 +171,27 @@ void annotationsShouldNotBeShownInXWiki10Syntax(TestUtils setup, TestReference t
         annotatableViewPage.simulateCTRL_M();
         annotatableViewPage.waitforAnnotationWarningNotification();
     }
+
+    @Test
```

Comment:
No order in those tests?

Reply:
Thanks. Fixed. I also added order number for existing tests, that were missing them as well.